site/src/styles/system.ts

```diff
@@ -37,6 +37,7 @@ export const {
       xxl: '2.5rem',
     },
     space: {
+      auto: 'auto',
```

Comment:
By adding `auto` is a value, we can reference it in `space` scale props as `$auto`.